libs/api_common/device_helper.cpp

```diff
@@ -71,21 +71,21 @@ clean_up_device_handle()
 {
```

Comment:
initialize_device_handle() initializes both _sync_device_handle and _async_device_handle.
Shouldn't clean_up_device_handle() then clean up both?

Reply:
@Alan-Jowett Still waiting for an answer to this comment.

Reply:
Fixed in: https://github.com/microsoft/ebpf-for-windows/pull/3315/commits/9b0bac9741498df94fa117e9f33832da67daa65f